assets/javascripts/initializers/communitarian.js.es6

```diff
@@ -1,13 +1,13 @@
 import { withPluginApi } from "discourse/lib/plugin-api";
 
 function initializeCommunitarian(api) {
-  console.log("Communitarian plugin initialized")
+  console.log("Communitarian plugin initialized");
 }
 
 export default {
   name: "communitarian",
 
   initialize() {
     withPluginApi("0.8.31", initializeCommunitarian);
-  }
+  },
```

Comment:
is a comma really needed here?

Reply:
no, this comma does not affect anything, it's just that I have auto-formatting configured this way